packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,232 @@
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_provisioningstatus_enum OWNER TO postgres;
+    
+    CREATE TYPE public.portfolio_status_enum AS ENUM (
+        'not_started',
+        'in_progress',
+        'failed',
+        'complete'
+    );
+    
+    ALTER TYPE public.portfolio_status_enum OWNER TO postgres;
+    
+    CREATE TYPE public.task_order_filescanstatus_enum AS ENUM (
+        'pending',
+        'accepted',
+        'rejected'
+    );
+    
+    ALTER TYPE public.task_order_filescanstatus_enum OWNER TO postgres;
+    
+    SET default_tablespace = '';
+    SET default_table_access_method = heap;
+    
+    CREATE TABLE public.application (
+        id uuid DEFAULT public.uuid_generate_v4() NOT NULL,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" public.application_provisioningstatus_enum DEFAULT 'not_started'::public.application_provisioningstatus_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        name character varying(100) NOT NULL,
+        description character varying(300),
+        "portfolioId" uuid
+    );
+    
+    ALTER TABLE public.application OWNER TO postgres;
+    
+    CREATE TABLE public.clin (
+        id uuid DEFAULT public.uuid_generate_v4() NOT NULL,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "clinNumber" character varying(4) NOT NULL,
+        "idiqClin" character varying NOT NULL,
+        "totalClinValue" money NOT NULL,
+        "obligatedFunds" money NOT NULL,
+        "popStartDate" date NOT NULL,
+        "popEndDate" date NOT NULL,
+        "taskOrderId" uuid
+    );
+    
+    ALTER TABLE public.clin OWNER TO postgres;
+    
+    COMMENT ON COLUMN public.clin."clinNumber" IS 'contract line item number from task order, 0001 through 9999';
+    COMMENT ON COLUMN public.clin."idiqClin" IS 'indefinite-delivery, indefinite-quantity CLIN specific to JWCC';
+    COMMENT ON COLUMN public.clin."totalClinValue" IS 'full amount requested by customer from USG for project or contract';
+    COMMENT ON COLUMN public.clin."obligatedFunds" IS 'amount already disbursed by USG for project or contract to be spent during CLIN POP';
+    COMMENT ON COLUMN public.clin."popStartDate" IS 'start of POP during which funds can be spent';
+    COMMENT ON COLUMN public.clin."popEndDate" IS 'end of POP during which funds can be spent';
```

Comment:
It looks like modifying comments in TS/models _will_ require a TypeORM migration in order to update which gives me pause about whether so many fields should have them. Will this result in a significant maintenance burden? Does having these comments in the database provide value? The answer very well may be "No" and "Yes" respectively but it feels worth considering.

Reply:
Removed in 3130a36b464102d41d157d051b8c705b1acdd983